packages/driver/src/dom/visibility.js

```diff
@@ -123,52 +119,196 @@ const elHasVisibilityHidden = ($el) => {
   return $el.css('visibility') === 'hidden'
 }
 
-const numberRegex = /-?[0-9]+(?:\.[0-9]+)?/g
-// This is a simplified version of backface culling.
+const elIsHiddenByTransform = ($el) => {
```

Comment:
Let's rename this function to fit its new behavior better. It used to return a boolean, but now it returns a string, so the `IsHidden` part doesn't really work anymore.